.cicd/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. $HELPERS_DIR/execute.sh
+
+execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == Darwin ]]; then
+
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
+
+    cd $BUILD_DIR
+    execute ccache -s
+    if [[ $ENABLE_BUILD == true ]] || [[ $TRAVIS == true ]]; then
+        [[ $TRAVIS == true ]] && execute mkdir -p wasms # Travis doesn't have git access to clone the private repo (Temporary)
+        execute cmake -DCMAKE_BUILD_TYPE=Release ..
+        execute make -j$JOBS
+    fi
+    if [[ $BUILDKITE == true ]]; then
+        [[ $ENABLE_TEST == true ]] && execute $MAC_TEST || true
+    elif [[ $TRAVIS == true ]]; then
+        echo "Tests disabled until wasms repo goes public"
+    fi
+
+else # Linux
+
+    MOUNTED_DIR='/workdir'
+    ARGS=${ARGS:-"--rm -v $(pwd):/workdir"}
+
+    function append-to-commands() {
+        [[ ! -z $COMMANDS ]] && export COMMANDS="$COMMANDS && $@" || export COMMANDS="$@"
+    }
+
+    . ./$HELPERS_DIR/docker-hash.sh
+
+    BUILD_COMMANDS="cd $MOUNTED_DIR/build && cmake -DCMAKE_TOOLCHAIN_FILE=$MOUNTED_DIR/.cicd/helpers/clang.make -DENABLE_TESTS=ON -DCMAKE_BUILD_TYPE=Release .. && make -j$JOBS"
+    TEST_COMMANDS="cd $MOUNTED_DIR/build && ctest -j$JOBS --output-on-failure -T Test"
```

Comment:
Let's hard-code `BUILD_COMMANDS` and `TEST_COMMANDS` as we do not expect these to change, and they are not meant to be configurable. Jerstern identified one of our primary goals as being to eliminate these types of variables.